editor/components/upload/upload.ts

```diff
@@ -17,7 +17,7 @@ export class Upload {
 
     handleFileChange(event: Event) {
         let files = (event.target as HTMLInputElement).files;
-        let file: File = files && files[0];
+        let file = files && files[0];
```

Comment:
Well, there is several point of view here. One might argue that as long as the type inference does the job, there is no need for us to do it, right? Yet, I myself usually prefere add the annotation type anyway if it helps understanding what the code is actually doing. Here, for instance, the fact that the variable can be `null` would be more obvious if it was actually written (with the `?` notation, if I understood correctly).

One might also argue that it is straightforward and therefore useless. I think your judgement is better than mine on this and just wanted to raise my point.